specification/cognitiveservices/data-plane/Face/.gitignore

```diff
@@ -0,0 +1,2 @@
+out/**
```

Comment:
hmm. don't include local `.gitignore ` files in the PR. You can keep them local, but we really don't need them in the master repo.